governance/polls/Vault Compensation Denomination - April 13, 2020.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Vault Compensation Denomination Poll - April 13, 2020
+summary: Signal whether to provide compensation to vault holders in the vault's token (ETH/BAT) or in DAI
+discussion_link: https://forum.makerdao.com/t/1822
+poll_rules: The voter may select to vote for one of the poll options or they may elect to abstain from the poll entirely
+options:
+   0: Abstain
+   1: ETH or BAT (i.e., the asset that was liquidated) 
+   2: DAI
+
+---
+# Vault Compensation Denomination Poll - April 13, 2020
+
+The Maker Foundation Interim Governance Facilitator has placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of the community which the community can use to signal whether to provide compensation to vault holders in the vault's token (ETH/BAT) or in DAI
+
+This Governance Poll ([FAQ](https://community-development.makerdao.com/makerdao-scd-faqs/scd-faqs/governance)) will be active for 7 days beginning on Monday, April 13 at 4 PM UTC.
+
+## Review
+
+[An initial poll on whether to compensate vault holders has passed.](https://vote.makerdao.com/polling-proposal/qmwfvvguaf8rz8xwgv2cqnzzt9t5h6epzh17qmk2ue99y4)
+
+This poll poses the choice of which token will be used to compensate, namely ETH/BAT or DAI. When vaults are liquidated, assets are sold for DAI to repay the debt. Any remaining assets are returned to the vault. In brief, in ordinary circumstances, vault holders would receive the type of asset that was originally held in the vault.
+
+Arguments for compensation in ETH/BAT:
+- Least surprising to vault owners
+- Avoids the risk that vault owners feel cheated, compromising the objectives of compensation: happy users, no distortion of symmetry, positive impact on reputation, and future mass adoption.
+
+Arguments for compensation in DAI:
+- Most likely, both BAT and ETH will have appreciated since the liquidation event. Therefore, a nominal 100% payout will almost certainly result in less value than would be obtained by payment in ETH/BAT. However, the payout amount could be adjusted to approximate the amount of appreciation in the planned subsequent poll.
```

Comment:
```suggestion
- Most likely, the price of both BAT and ETH will change since the liquidation event. Therefore, a nominal 100% payout will almost certainly result in a different value than would be obtained by payment in ETH/BAT. However, the payout amount could be adjusted to approximate the amount of appreciation in the planned subsequent poll.
```